### PR TITLE
projects: add crud commands

### PR DESCRIPTION
**Pull request by davidhewitt**
This PR adds commands to manage projects via the CLI.

### CODE REVIEW

File: cli/src/commands/create/project.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    #[structopt(long = "title")]
+    /// Set the title of the new project
+    title: Option<String>,
+
+    #[structopt(long = "description")]
+    /// Set the description of the new project
+    description: Option<String>,
+
+    #[structopt(long = "user-ids", required = true)]
+    /// Set the description of the new project
```
Comment (tommilligan) [inline cli/src/commands/create/project.rs:22]: nit: bad comment

File: cli/src/commands/create/project.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    user_ids: Vec<UserId>,
```
Comment (tommilligan) [inline cli/src/commands/create/project.rs:23]: This could also be usernames I guess in future, but agree user ids are fine for now

**Review by tommilligan — APPROVED** [review]
Amazing, thanks for adding this :pray:

**Review by mcobzarenco — APPROVED** [review]
Thanks so much for adding these commands, super useful :100:

**Pull request merged**